pkg/controller/ehpa/effective_hpa_controller.go

```diff
@@ -20,6 +20,7 @@ import (
 
 	autoscalingapi "github.com/gocrane/api/autoscaling/v1alpha1"
 	predictionapi "github.com/gocrane/api/prediction/v1alpha1"
+
```

Comment:
nit: why a new section?

Reply:
In newest code standard, the import parts are system imports, community imports and crane apis and crane imports.